### PR TITLE
defend against non-string device tag values

### PR DESCRIPTION
**Pull request by MaxBittker**
fixes JAVASCRIPT-22M

root cause is an object being passed to this function under rare conditions I don't yet understand. this will keep the page usable in the meantime 
screenshot 2017-05-11 16 45 07
https://sentry.io/share/issue/31313237362e323731303733313434/

### CODE REVIEW

File: src/sentry/static/sentry/app/utils/deviceNameMapper.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import iOSDeviceList from 'ios-device-list';
 
 export default function(model) {
-  if (!model) {
+  if (!model || typeof model !== 'string') {
```
Comment (mattrobenolt) [inline src/sentry/static/sentry/app/utils/deviceNameMapper.jsx]: We use the `isString` from `_` elsewhere it seems becuase I'm pretty sure this doesn't cover all the cases.

Comment (MaxBittker) [inline src/sentry/static/sentry/app/utils/deviceNameMapper.jsx]: _.👍

**Comment by MaxBittker** [comment]
@mitsuhiko if the fact that a handful of people had objects as values on these tags doesn't bother you, I want to ship this today

**Review by LewisJEllis — APPROVED** [review]
LGTM

**Pull request merged**